sdk/storage/azure_storage_blob/README.md

```diff
@@ -1,3 +1,140 @@
-# sdk/storage/azure_storage_blob
+# Azure Storage Blobs client library for Rust
 
-TODO TODO TODO
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data, such as text or binary data.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [REST API documentation] | [Product documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Storage Blobs client library for Rust with [cargo]:
+
+```sh
+cargo add azure_storage_blob
+```
+
+### Prerequisites
+
+* You must have an [Azure subscription] and an [Azure storage account] to use this package.
+
+
+
+### Create a storage account
+If you wish to create a new storage account, you can use the
+[Azure Portal], [Azure PowerShell], or [Azure CLI]:
+```bash
+# Create a new resource group to hold the storage account -
+# if using an existing resource group, skip this step
+az group create --name my-resource-group --location westus2
+
+# Create the storage account
+az storage account create -n my-storage-account-name -g my-resource-group
+```
+
+#### Authenticate the client
+
+In order to interact with the Azure Blobs Storage service, you'll need to create an instance of a client, `BlobClient`, `BlobContainerClient`, or `BlobServiceClient`. The [Azure Identity] library makes it easy to add Azure Active Directory support for authenticating Azure SDK clients with their corresponding Azure services:
```

Comment:
"Active Directory" is the old term. Please change to Entra Id here and in code comments below.